### DISCUSSION TITLE
APE error after reloadscripts with OO script

### DISCUSSION THREAD

**Discussion by cressidagp** [arcemu/arcemu#461] [Q&A] [answered]
Hi there,

I discover than using reloadscripts command will return an error if there are python scripts writed with paradigm OO

"scriptMgr its trying to register a script for creature id 10616 even if theres already one for that creature"

Its also happen with rabbit script.

¿its a bug or a necessary consequence?

**Comment by dfighter1985** [comment]
If you test it you will see that the Python script gets reloaded fine.
What Arcemu is complaining about is that the script factory method is already registered.
You should not worry about this.

**Comment by dfighter1985** [comment, marked as answer]
e6d130b fixed this.
